src/components/RootApp/RootApp.tsx

```diff
@@ -110,19 +108,17 @@ const RootApp = memo((props: RootAppProps) => {
     <HistoryRouter history={chromeHistory as unknown as HistoryRouterProps['history']} basename={isBeta() ? '/beta' : '/'}>
       <SegmentProvider activeModule={activeModule}>
         <FeatureFlagsProvider>
-          <IDPChecker>
```

Comment:
The IDP checker is required for normal environments. If it's problematic in fedramp it can be disabled, but it has to stay in the rest of the environments.

Reply:
I did local build and this seems to be not needed in stage either, keith told me it was added for fr in the first place

Reply:
OK. If Keight says that then its probably true.